tensorboard/plugins/pr_curve/summary.py

```diff
@@ -58,51 +53,45 @@ def op(
 
   Args:
     tag: A tag attached to the summary. Used by TensorBoard for organization.
-    labels: The ground truth values, a `Tensor` whose dimensions must match
-        `predictions`. Should be of type `bool`.
-    predictions: A floating point `Tensor` whose values are in the range
-        `[0, 1]`. Shape is arbitrary if `num_classes=1`; otherwise, the
-        last dimension should be exactly `num_classes`.
+    labels: The ground truth values. A Tensor of `bool` values with arbitrary
+        shape.
+    predictions: A float32 `Tensor` whose values are in the range `[0, 1]`.
+        Dimensions must match those of `labels`.
     num_thresholds: Number of thresholds, evenly distributed in `[0, 1]`, to
         compute PR metrics for. Should be `>= 2`. This value should be a 
         constant integer value, not a Tensor that stores an integer.
-    weights: Optional; If provided, a `Tensor` that has the same dtype as,
-        and is broadcastable to, `predictions`.
-    num_classes: Optional; Used when `predictions` is a multi-class classifier.
-    display_name: The name displayed atop this PR curve in TensorBoard. The
-        display_name is optional. `tag` will be used in its absence.
-    description: Not yet supported; do not use. (Eventually: Optional
-        long-form description for this summary. Markdown is supported.)
+    weight: Optional; A float or scalar float32 `Tensor`. Individual
+        counts are multiplied by this value.
+    display_name: Optional; The name displayed atop this PR curve in
+        TensorBoard. `tag` will be used in its absence.
+    description: Optional; A text description describing this summary. Appears
+        in TensorBoard near the corresponding visualization.
```

Comment:
I copied over the docs for `display_name` and `description`. That also reminded me to add a `collections` argument, so I did that.